code/modules/mob/living/carbon/human/human.dm

```diff
@@ -368,8 +368,13 @@
 			var/pocket_id = (pocket_side == "right" ? slot_r_store : slot_l_store)
 			var/obj/item/pocket_item = (pocket_id == slot_r_store ? r_store : l_store)
 			var/obj/item/place_item = usr.get_active_hand() // Item to place in the pocket, if it's empty
-
 			var/delay_denominator = 1
+			var/has_pickpocket = 0
```

Comment:
This is still pretty snowflakey...it'd be better if you made this var the gloves' var,and then checif it exists k